src/Json.Schema.ToDotNet/ClassGenerator.cs

```diff
@@ -291,16 +292,16 @@ protected override AttributeSyntax[] GeneratePropertyAttributes(string propertyN
             {
                 // We want to add a DefaultValue attribute, but we can only do that if the
                 // default value specified in the schema can be represented by a literal
-                // (a compile-time constant like 42 or "Don't panic"). We can't do it if the
-                // default value must be calculated at runtime, for example, an empty array.
-                LiteralExpressionSyntax literalExpression = GetLiteralExpressionForValue(defaultValue);
-                if (literalExpression != null)
+                // (a compile-time constant like 42 or "Don't panic" or Color.Red). We can't do it if
+                // the default value must be calculated at runtime, for example, an empty array.
+                ExpressionSyntax expression = GetExpressionForValue(defaultValue, propertyType);
```

Comment:
>ExpressionSyntax [](start = 16, length = 16)

The expression for the default value is no longer a "literal expression" because to Roslyn, `Color.Green` is not a "literal" -- Roslyn considers it a "simple member access expression."